src/main/java/org/radarbase/appserver/controller/GithubEndpoint.java

```diff
@@ -26,16 +26,13 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestParam;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
 import radar.spring.auth.common.Authorized;
 
 import java.io.IOException;
 import java.net.MalformedURLException;
 
-@CrossOrigin
+@CrossOrigin(methods = RequestMethod.GET)
```

Comment:
Actually, apologies I did not realise before, but @CrossOrigin by default only adds methods that are defined by mapping annotations in the class, so you don't need to specify these.

Reply:
Ok I see. I've reverted the changes.